src/Resources/contao/classes/DataContainer.php

```diff
@@ -1027,4 +1064,38 @@ protected function getPickerInputField($value, $attributes='')
 	 * @throws \Exception
 	 */
 	abstract protected function save($varValue);
+
+
+	/**
+	 * Create a data container object in the picker environment
+	 *
+	 * @return DataContainer
+	 */
+	private function getDataContainer()
+	{
+		$do = \Input::get('do');
```

Comment:
But custom checks might use `table` too.
Not setting the `table` parameter could lead to confusing bugs IMO. I think we should set it too.